packages/eslint-plugin/src/rules/no-floating-promises.ts

```diff
@@ -110,7 +133,10 @@ export default util.createRule<Options, MessageId>({
           } else {
             context.report({
               node,
-              messageId: 'floating',
+              messageId:
+                specialReason === 'nonFunctionHandler'
+                  ? 'floatingUselessRejectionHandler'
+                  : 'floating',
```

Comment:
Added several test cases. Hoping the next coverage report comes back clean!

(to clarify - the default is `ignoreVoid: true`, so the ones that were added are the ones with `ignoreVoid: false`)

Reply:
Awesome, thanks!

In case it's helpful, you can always view the coverage report locally. It's just [Jest with `coverage`](https://jestjs.io/docs/cli#--coverageboolean), outputting to `packages/*/coverage/lcov-report/index.html`.